api/plugin_helpers.go

```diff
@@ -40,7 +40,7 @@ const (
 // path matches that path or not (useful specifically for the paths that
 // contain templated fields.)
 var sudoPaths = map[string]*regexp.Regexp{
-	"/auth/token/accessors/":                        regexp.MustCompile(`^/auth/token/accessors/?$`),
+	"/auth/token/accessors":                         regexp.MustCompile(`^/auth/token/accessors/?$`),
```

Comment:
I think it is worth adding these modified paths to the `TestIsSudoPath` in [here](https://github.com/hashicorp/vault/blob/00295d2bf5b6cb67f2a939bd602b780aa63831cb/api/plugin_helpers_test.go#L8) just to keep track of them.

Reply:
I'm happy to add a couple more test cases focussed around list-only endpoints, but I'm not sure what you mean by "keep track of them" - the existing test cases only test some represenatative examples of sudo paths to confirm the code is correct, so I don't think I should copy/paste the same tests for `auth/token/accessors` and `sys/config/ui/headers`, which would be identical apart from the constant string.

Reply:
It would be great to add some tests around list-only endpoints.

Reply:
Done!

Reply:
Great! Thank you!